sdk/file_sdk/Common/tag.h

```diff
@@ -44,18 +43,25 @@ class Tag final
   const std::string& GetLabelId() const { return mLabelId; }
   const std::string& GetLabelName() const { return mLabelName; }
   const std::string& GetLabelParentId() const { return mLabelParentId; }
+  const std::string& GetOwner() const { return mOwner; }
+  const bool GetEnabled() const { return mEnabled; }
+  const std::string& GetSiteId() const { return mSiteId; }
+  const std::string& GetSetTime() const { return mSetTime; }
+  const Method GetMethod() const { return mMethod; }
+  const std::vector<ExtendedProperty>& GetExtendedProperties() const { return mExtendedProperties; }
 
 private:
   // TODO: Add tests for these methods.
   static std::string GetBackwardCompatibleKey(
-    const std::vector<std::pair<std::string, std::string>>& properties,
-    const std::string& id,
-    const std::string& key,
-    const std::vector<std::string>& backwardCompatibleKeys);
+      const std::vector<std::pair<std::string, std::string>>& properties,
+      const std::string& id,
+      const std::string& key,
+      const std::vector<std::string>& backwardCompatibleKeys);
   static std::string GetDataForKey(
-    const std::vector<std::pair<std::string, std::string>>& properties, const std::string& key);
+      const std::vector<std::pair<std::string, std::string>>& properties, const std::string& key);
+public:
   static std::vector<std::string> GetAllIds(

```

Comment:
remove this

Reply:
why? i think its used in UPE so we will need it public

---
In reply to: [138032241](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r138032241) [](ancestors = 138032241)

Reply:
you made your fields public

---
In reply to: [138036569](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r138036569) [](ancestors = 138036569,138032241)